common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -434,6 +434,18 @@ def validate(self):
         validation = super(LibraryContentDescriptor, self).validate()
         if not isinstance(validation, StudioValidation):
             validation = StudioValidation.copy(validation)
+        library_tools = self.runtime.service(self, "library_tools")
+        if not (library_tools and library_tools.can_use_library_content(self.course_id)):
```

Comment:
This is causing the test failures. Should be self.runtime.course_id, or even better (since XBlocks aren't really supposed to know about "courses"), just pass in 'self' and let the can_use_library_content method get the course_id from that using .runtime.course_id or .scope_ids.usage_id.course_id